pkg/scheduler/scheduler.go

```diff
@@ -303,7 +303,9 @@ func New(client clientset.Interface,
 			VolumeBinder: volumeBinder,
 		})
 	}
-	registry.Merge(options.frameworkOutOfTreeRegistry)
+	if err := registry.Merge(options.frameworkOutOfTreeRegistry); err != nil {
```

Comment:
this seem like a valid error. Could you please add a test case in `TestSchedulerCreation`?
Or should it be compatibility test @ahg-g?

Reply:
> this seem like a valid error. Could you please add a test case in `TestSchedulerCreation`?
> Or should it be compatibility test @ahg-g?

i add a test case in `TestSchedulerCreation`, please have a review on this, thanks.